### ISSUE TITLE
[Feature] Support LibreJS

### ISSUE THREAD

**Issue by asddsaz** [PrivateBin/PrivateBin#392]
**Description**: Make PrivateBin LibreJS Compliant
More Info: https://www.gnu.org/software/librejs/free-your-javascript.html

It appears to flag the following scripts?

- https://privatebin.net/js/kjua-0.1.2.js

- https://privatebin.net/js/base64-2.4.5.js

- https://privatebin.net/js/sjcl-1.0.7.js

- https://privatebin.net/js/rawdeflate-0.5.js

- https://privatebin.net/js/rawinflate-0.3.js

- https://privatebin.net/js/bootstrap-3.3.7.js

- https://privatebin.net/js/prettify.js

- https://privatebin.net/js/showdown-1.8.6.js

- https://privatebin.net/js/purify-1.0.7.js

- https://privatebin.net/js/privatebin.js

**Comment by rugk** [comment]
Yeah, I guess we just have to add some comments to the JS files. However, the problem here would be, that [the used zlib license](https://github.com/PrivateBin/PrivateBin/blob/master/LICENSE.md#zliblibpng-license-for-privatebin) is not [listed as a magnet link](https://www.gnu.org/software/librejs/free-your-javascript.html#magnet-link-license).
